src/styles/blocks/container.scss

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: 1024px;
+  padding: 50px 40px;
```

Comment:
now it looks like you have `container` bem block and add external styles for it

you could do something like that
```
<main class="container">
      <div  class="container__catalog">
        <div class="container__card">